compiler/test/dotty/tools/dotc/BootstrappedOnlyCompilationTests.scala

```diff
@@ -129,6 +129,7 @@ class BootstrappedOnlyCompilationTests extends ParallelTesting {
       compileFilesInDir("tests/run-staging", withStagingOptions),
       compileFilesInDir("tests/run-custom-args/tasty-inspector", withTastyInspectorOptions),
       compileDir("tests/run-custom-args/tasty-interpreter", withTastyInspectorOptions),
+      compileFile("tests/run-custom-args/i8163.scala", withTastyInspectorOptions),
```

Comment:
The test should be moved to `tests/run-custom-args/tasty-inspector/i8163.scala` an this line won't be necessary.